week-2/debug/2.js

```diff
@@ -1,8 +1,8 @@
 // Predict and explain first...
 
-const num = 103;
+//const num = 103;
 
-function getLastDigit() {
+function getLastDigit(num) {
```

Comment:
You've corrected the function here but you also need to add a brief explanation for why it didn't work in the first place